package.json

```diff
@@ -48,10 +48,13 @@
     "@hashicorp/react-inline-svg": "^6.0.3",
     "@hashicorp/react-min-100-layout": "^2.0.1",
     "@hashicorp/react-product-downloads-page": "^2.5.2",
+    "@hashicorp/react-search": "^6.4.0",
     "@hashicorp/react-section-header": "^5.0.4",
     "@hashicorp/react-subnav": "^9.1.3",
     "@hashicorp/react-vertical-text-block-list": "^7.0.0",
     "@hashicorp/remark-plugins": "^3.3.1",
+    "@reach/popover": "^0.16.2",
+    "@reach/portal": "^0.16.2",
```

Comment:
Added for `popover` component, and by extension the `dev-popover` used for the demo note on Search.